src/components/slider/slider.js

```diff
@@ -119,6 +119,12 @@ class Slider extends mixin(createComponent, initComponentBySearch, eventedState)
         }
       }
       if (type === 'mousemove' || type === 'click') {
+        if (type === 'click') {
+          document.querySelector(this.options.selectorThumb).classList.add('bx--slider__thumb--clicked');
```

Comment:
@tw15egan Did you mean `this.element.querySelector(this.options.selectorThumb)` (thumb in this slider instance)?

Reply:
Yes, that's exactly what I meant 😄  Haven't worked in this javascript in awhile! Updating now.

Reply:
Cool thanks @tw15egan!